go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce
 	github.com/docker/go-units v0.4.0
-	github.com/evanphx/json-patch v0.0.0-20200808040245-162e5629780b
+	github.com/evanphx/json-patch/v5 v5.0.0
```

Comment:
The older hash based version references v5.1.0. See https://github.com/evanphx/json-patch/commit/162e5629780b. Can you please set the minimum version to v5.1.0 so that the same code is used.